.github/workflows/update_documentation.yml

```diff
@@ -0,0 +1,56 @@
+on:
+  workflow_dispatch:
+  pull_request_review:
+    types: [submitted]
+
+name: update_documentation
+
+jobs:
+  update_documentation:
+    if: github.event.review.state == 'approved'
+    runs-on: ubuntu-20.04
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/cache@v2
+        with:
+          path: ~/.local/share/renv
+          key: ${{ runner.os }}-renv-${{ hashFiles('**/renv.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-renv-
+
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v1
+        with:
+          r-version: 3.6
+
+      - name: Query dependencies
+        run: |
+          install.packages('remotes')
+          saveRDS(remotes::dev_package_deps(dependencies = TRUE), ".github/depends.Rds", version = 2)
+          writeLines(sprintf("R-%i.%i", getRversion()$major, getRversion()$minor), ".github/R-version")
+        shell: Rscript {0}
+
+      - name: Install system dependencies
+        if: runner.os == 'Linux'
+        run: |
+          while read -r cmd
+          do
+            eval sudo $cmd
+          done < <(Rscript -e 'writeLines(remotes::system_requirements("ubuntu", "20.04"))')
+
+      - name: Install dependencies
+        run: |
+          if (!requireNamespace("renv", quietly = TRUE)) install.packages("renv")
+          renv::restore()
+        shell: Rscript {0}
+
+      - name: Update documentation
+        run: |
+          git config --local user.email "actions@github.com"
+          git config --local user.name "GitHub Actions"
+          Rscript -e "devtools::document()"
+          git add .
```

Comment:
I think this adds only the files in the current directory. `git add -A` should add all files.
However, maybe we should use `git add -f NAMESPACE man/*.Rd` and add `NAMESPACE` and `man` to the .gitgnore file.

Reply:
Do you think adding to `.gitignore` is necessary? After all, the reason for adding this workflow is that people forget to run `devtools::document()` and commit the created files. I agree on specifically adding only `NAMESPACE` and `man/*.Rd`, though.

Reply:
I think adding the files to .gitignore makes the development easier. Otherwise, the files are listed as modified, which may cause confusion.
What is the advantage of not adding them to .gitignore?

Reply:
Fair enough, I've made the change.

Reply:
What about `NAMESPACE`? Shouldn't it be added to .gitignore as well?

Reply:
Indeed,